lib/main.dart

```diff
@@ -16,6 +17,8 @@ main() async {
   db = AppDatabase(constructDb());
   setupLocator();
   await locator<Preferences>().loadData();
+  final account = AccountController();
```

Comment:
now i know this is only temporary, but still we should move every *Controller class to the locator for better access and for not having a shit ton of instances all around the app

Reply:
I'll add the AccountController to it as well